### PR TITLE
Populate CEPs from `conda/ceps`

### PR DESCRIPTION
**Pull request by jaimergp**
### Description

- [X] If I have added a new page to `learn/` or `community/`, I have added it to the corresponding `_sidebar.json` file.
- Closes https://github.com/conda/ceps/issues/76
- Merge once https://github.com/conda/ceps/pull/96 goes in

### CODE REVIEW

File: bin/populate-ceps [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+HERE=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+rm "${HERE}/../community/ceps/*.md" || mkdir -p "${HERE}/../community/ceps/"
+pushd $(mktemp -d)
+curl -L https://github.com/jaimergp/ceps/archive/editorial-updates.tar.gz -o ceps.tar.gz
+tar -xf ceps.tar.gz
+cd ceps-editorial-updates
```
Comment (jaimergp) [inline bin/populate-ceps:10]: - TODO: Apply this revert once https://github.com/conda/ceps/pull/96 is merged.

```suggestion
curl -L https://github.com/conda/ceps/archive/main.tar.gz -o ceps.tar.gz
tar -xf ceps.tar.gz
cd ceps-main
```

**Comment by jaimergp** [comment]
Ready for review! cc @conda/steering-council

**Review by beeankha — APPROVED** [review]
This is great! ✨

**Review by kathatherine — APPROVED** [review]
Awesome!

**Comment by jezdez** [comment]
@jaimergp Thinking about it more, I think the CEPs might actually be better fitting in the learn section than the community section, given that CEPs are technical specification and not related to community management or organization.

**Pull request merged**